CHANGELOG.md

```diff
@@ -47,7 +47,9 @@ for currentStopLossLimitActivation
 - :tada: **Enhancement** Add constructors with parameters to allow custom implementation of ReportGenerators in BacktestExecutor
 - :tada: **Enhancement** Added license checker goal on CI's pipeline
 - :tada: **Enhancement** Added source format checker goal on CI's pipeline
-
+- :tada: **Enhancement** Added DateTimeIndicator a new Indicator for dates.
```

Comment:
This feature was postponed to version 0.14 so, please update the changelog info.

Reply:
Done